apps/emqx_durable_storage/src/emqx_ds_message_storage_bitmask.erl

```diff
@@ -277,13 +277,13 @@ store(DB = #db{handle = DBHandle, cf = CFHandle}, MessageID, PublishedAt, Topic,
     Value = make_message_value(Topic, MessagePayload),
     rocksdb:put(DBHandle, CFHandle, Key, Value, DB#db.write_options).
 
--spec make_iterator(db(), emqx_ds:replay()) ->
+-spec make_iterator(db(), emqx_ds_replay:replay()) ->
```

Comment:
Sidenote: I think this module is currently pretty much empty, I think everything from `emqx_ds_replay` can be moved to `emqx_ds`